packages/manager/modules/octavia-load-balancer/src/create/controller.js

```diff
@@ -115,8 +134,49 @@ export default class OctaviaLoadBalancerCreateCtrl {
       });
   }
 
+  onCancel() {
+    this.resetCurrentStep();
+    this.atInternet.trackClick({
+      name: TRACKING_LOAD_BALANCER_CREATION_CANCEL,
+      type: 'action',
+    });
+  }
+
   createLoadBalancer() {
-    // TODO: REMOVE THIS LINE
-    this.model.submit = true;
+    this.atInternet.trackClick({
+      name: TRACKING_LOAD_BALANCER_CREATION_SUBMIT,
+      type: 'action',
+    });
+
+    this.atInternet.trackClick({
+      name: `${TRACKING_LOAD_BALANCER_CREATION_SUBMIT_DETAIL}::${this.model.size.code}::${this.model.region.name}`,
+      type: 'action',
+    });
+
+    this.OctaviaLoadBalancerCreateService.createLoadBalancer(
+      this.projectId,
+      this.model.size,
+      this.model.region.name,
+      this.model.privateNetwork,
+      this.model.subnet,
+      this.gateways,
+      // null, // TODO: AFTER LISTENERS DEVELOPEMENT
+      this.model.loadBalancerName,
+    )
+      .then(() => {
+        this.atInternet.trackPage({
+          name: TRACKING_LOAD_BALANCER_CREATION_SUBMIT_SUCCESS,
+        });
+
+        this.$state.go('octavia-load-balancer', {
+          displayCreationBanner: true,
+        });
```

Comment:
**nitpick:** Avoid injecting `$state` into controller. Instead provide a callback from `$state.resolve` and use the callback function.